tmail-backend/mailbox/opensearch/src/main/java/com/linagora/tmail/mailbox/opensearch/TmailMailboxMappingFactory.java

```diff
@@ -341,7 +341,7 @@ private Map<String, Property> getSubjectFieds() {
 
         if (tmailOpenSearchMailboxConfiguration.subjectNgramEnabled()) {
             subjectFields.put(NGRAM, new Property.Builder()
-                .text(new TextProperty.Builder().analyzer(NGRAM_ANALYZER).searchAnalyzer(STANDARD).build())
+                .text(new TextProperty.Builder().analyzer(NGRAM_ANALYZER).searchAnalyzer(FILENAME_ANALYZER).build())
```

Comment:
cc @Arsnael I think we can improve the subject search a bit. Review welcome.

Reply:
Not sure if it's really necessary for the subject, I have nothing against it though